data/transactions/logic/eval.go

```diff
@@ -1678,6 +1845,67 @@ func opSubstring3(cx *evalContext) {
 	cx.stack = cx.stack[:prev]
 }
 
+func opGetBit(cx *evalContext) {
```

Comment:
if we're already adding get/set bit, can we add that functionality to support byte arrays as well ?